src/notifications/generate.ts

```diff
@@ -296,4 +298,11 @@ export const generateNotificationMap: Record<
       .avatarManySources(ctx.sources)
       .numTotalAvatars(ctx.total)
       .uniqueKey(ctx.post.metadataChangedAt?.toString()),
+  dev_card_unlocked: (builder) =>
+    builder
+      .icon(NotificationIcon.Bell)
+      .description(
+        'You can now generate your own DevCard to showcase your daily.dev achievements.',
+      )
+      .targetUrl(generateDevCard),
```

Comment:
We need to add `.uniqueKey(ctx.userIds[0])` here. Otherwise after the first notification, we will get duplicate error issue